requirements/dev.txt

```diff
@@ -10,7 +10,6 @@ freezegun==1.5.1
 isort==5.13.2
 mkdocs==1.6.1
 mkdocstrings[python]==0.26.1
-psycopg[binary]==3.2.2
```

Comment:
you need to add this back. I guess was deleted by accident. If you read the tests failing you can see that the psycopg library is missing.

Reply:
Thanks @m4ra! I erased that line because there was some versions problems with it! I upgraded the OS and now I'm installing Xcode dev tools again! I will add the line and let you know!